static/src/js/containers/TimelineContainer/TimelineContainer.js

```diff
@@ -51,11 +52,13 @@ export const TimelineContainer = (props) => {
     onToggleOverrideTemporalModal,
     onMetricsTimeline,
     onToggleTimeline,
-    isOpen
+    isOpen,
+    search: searchLocation
   } = props
 
   // Determine the collectionMetadata the timeline should be displaying
-  const isProjectPage = isPath(pathname, ['/projects'])
+  // Ensure that timeline does not appear on the `Saved Projects` page
+  const isProjectPage = isPath(pathname, ['/projects']) && (searchLocation.length > 0)
```

Comment:
What happened here?

Reply:
See screenshot #2 timeline in prod is showing up if navigating from non-empty project to the saved projects page